.github/workflows/release.yml

```diff
@@ -15,5 +15,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: cli/gh-extension-precompile@561b19deda1228a0edf856c3325df87416f8c9bd # v2.0.0
         with:
+          release_android: true
+          android_sdk_version: 30 # Android 11.0
```

Comment:
Using [Android SDK currently available](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#android) on GitHub-managed runners, we might want to change this version to 31/32/33/34/35 🤔